src/ServiceManagement/Common/Commands.ScenarioTest/Resources/ServiceManagement/ServiceManagementTests.ps1

```diff
@@ -912,6 +912,30 @@ function Test-MigrationAzureReservedIP
     Assert-AreEqual $removeReservedIP.OperationStatus "Succeeded"
 }
 
+<#
+.SYNOPSIS
+Tests New-AzureReservedIPWithTags
+#>
+function Test-AzureReservedIPWithIPTags
+{
+    # Setu
+    $name = getAssetName
+    $location = "West Central US"
```

Comment:
HI Mark,

This feature is not enabled in all regions.. it is infact enabled only in 1 region that is West Central US